model/Licensing/Properties/additionId.md

```diff
@@ -9,8 +9,9 @@ Provides a short, unique identifier to refer to a LicenseAddition.
 ## Description
 
 An additionId contains a human-readable, short-form identifier for a
-LicenseAddition. It may only include letters, numbers, period (".") and
-hyphen ("-") characters.
+LicenseAddition. It may only include letters, digits, periods (“.”) and
```

Comment:
Can you maybe avoid non-ascii characters, as they might confuse the parsing. 
```suggestion
LicenseAddition. It may only include letters, digits, periods (".") and
```

Reply:
It is touching undefined behavior that was never discussed. We have no idea if every documentation that we want to generate supports utf-8 in descriptions. Therefore I was suggesting to stay conservative, to have less issues down the road.

Our parsing and generation tooling is right now not strict enough so that it could catch such potential problems. And it is not keeping up.

Reply:
The following files currently are in UTF-8:

- ./Glossary.md: Unicode text, UTF-8 text, with very long lines (993)
- ./License.md: Unicode text, UTF-8 text, with very long lines (1977)
- ./README.md: Unicode text, UTF-8 text
- ./model/Software/Vocabularies/SBOMType.md: Unicode text, UTF-8 text
- ./model/Software/Properties/attributionText.md: Unicode text, UTF-8 text
- ./model/Software/Properties/declaredLicense.md: Unicode text, UTF-8 text
- ./model/Core/Properties/dataLicense.md: Unicode text, UTF-8 text

Reply:
The ones in the spec (inside `/model`) have to be fixed, then. We should have consistency, at the very least.